plugin/src/main/java/org/openrewrite/gradle/isolated/DefaultProjectParser.java

```diff
@@ -439,17 +449,21 @@ public void run(ResultsContainer results, ExecutionContext ctx) {
                                      result.getBefore().getSourcePath() + " to " +
                                      result.getAfter().getSourcePath() + " by:");
                     logRecipesThatMadeChanges(result);
+                    estimateTimeSaved = estimateTimeSavedSum(result, estimateTimeSaved);
                 }
                 for (Result result : results.refactoredInPlace) {
                     assert result.getBefore() != null;
                     logger.lifecycle("Changes have been made to " +
                                      result.getBefore().getSourcePath() +
                                      " by:");
                     logRecipesThatMadeChanges(result);
+                    estimateTimeSaved = estimateTimeSavedSum(result, estimateTimeSaved);
                 }
 
                 logger.lifecycle("Please review and commit the results.");
 
+                logger.lifecycle("Estimate time saved : {}", estimateTimeSaved);
```

Comment:
By default this will print something like `PT8H6M12.345S` which is perhaps not very user friendly. Would we want to make this more human readable?

Reply:
I think I like this quick no-library solution to a more readable form best: https://stackoverflow.com/a/40487511

Reply:
I made a function which is using string builder but I can use the way in your link.

Reply:
There is common library between Gradle and Maven plugins ?

Reply:
Let's opt for that four liner in the link above; that way it's small enough to not have to share that between the two plugins.

Reply:
It is on the pull request.